dashboard/test/ui/runner.rb

```diff
@@ -393,7 +402,7 @@ def how_many_reruns?(test_run_string)
       return 1
     elsif $options.magic_retry
       # ask saucelabs how flaky the test is
-      flakiness = TestFlakiness.test_flakiness[test_run_string]
+      flakiness = flakiness_for_test(test_run_string)
```

Comment:
Just noticed: Is the comment `# ask saucelabs how flaky the test is` inaccurate now, since we're actually reading a cached flakiness and we ask saucelabs earlier in the process?